core/pkg/service/ofrep/models.go

```diff
@@ -95,6 +95,9 @@ func EvaluationErrorResponseFrom(result evaluator.AnyValue) (int, EvaluationErro
 	case model.ParseErrorCode:
 		payload.ErrorCode = model.ParseErrorCode
 		payload.ErrorDetails = fmt.Sprintf("error parsing the flag `%s`", result.FlagKey)
+	case model.FlagDisabledErrorCode:
+		payload.ErrorCode = model.GeneralErrorCode
```

Comment:
This should be a flag not found error

Reply:
Thanks and done. Yeah this match with flagd rpc evaluation 👍